AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -437,8 +437,8 @@ public class ConstructorIO: CIOSessionManagerDelegate {
      constructorIO.trackBrowseResultsLoaded(filterName: "Category", filterValue: "Snacks", resultCount: 674, resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```

Comment:
@mocca102 Forgot to mention this previously, but could we update the code example here and as well as the README example? Thanks!

Reply:
Great catch!

Reply:
Thanks! I'm not sure if we should surface it as another example with "(supported in vX.X.X and above)", but I'll leave that to you. 😄